ext/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+name = "django-stubs-ext"
+# NB! For clarity, keep version major.minor.patch in sync with django-stubs.
+# It's fine to skip django-stubs-ext releases, but when doing a release, update this to newest django-stubs version.
+version = "5.2.0"
+description = "Monkey-patching and extensions for django-stubs"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+maintainers = [
+  { name = "Marti Raudsepp", email = "marti@juffo.org" },
+]
+authors = [
+  { name = "Simula Proxy" },
+]
+requires-python = ">=3.10"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Typing :: Typed",
+    "Framework :: Django",
+    "Framework :: Django :: 5.0",
```

Comment:
The table in the readme only expresses partial support for 5.1 for stubs v5.2:

```suggestion
```

Reply:
We are still testing against 5.0, 5.1 and 5.2 in ci so I believe I should amend the readme instead ? 

https://github.com/typeddjango/django-stubs/blob/master/.github/workflows/test.yml#L136

Reply:
Fair.